src/OpenLoco/Map/BuildingTile.cpp

```diff
@@ -55,6 +55,25 @@ namespace OpenLoco::Map
         return town;
     }
 
+    // 0x0046902E
+    void sub_46902E(const Pos2& pos)
```

Comment:
This should be under TileManager i think. Make it `TileManager::removeSurfaceIndustry` or something like that.

Reply:
I started with it in `TileManager.cpp` and moved it here because I needed to use `sub_497DC1` and it was easier here. How can I use sub_497DC1 in `TileManager.cpp`? I don't know if I should add it to BuildingElement like the other function in this file.

Reply:
sub_497DC1 should probably move to town manager.